Application/EdFi.Ods.Common/Conventions/ExtensionsConventions.cs

```diff
@@ -77,34 +74,13 @@ public static string GetExtensionClassAssemblyQualifiedName(Type edFiStandardTyp
             {
                 //If standard type exists in profile assembly, use edFiStandardType assembly name.
```

Comment:
Is this comment still relevant to be updated or can be removed?

Reply:
I cleaned this up.